frontends/p4/validateParsedProgram.cpp

```diff
@@ -201,13 +201,8 @@ void ValidateParsedProgram::postorder(const IR::EntriesList* l) {
                 "%1%: invalid initializer. Table initializers must be constant.", l);
 }
 
-/// Switch statements are not allowed in actions.
```

Comment:
Is this related to https://github.com/p4lang/p4-spec/pull/945?

Reply:
There is a little change related to that.
But, as I said in the spec PR, because you can't call tables in actions you can't use the old switch statements anyway.
So I think this check does not change anything really.